Lib/test/libregrtest/utils.py

```diff
@@ -264,6 +264,13 @@ def clear_caches():
         for f in typing._cleanups:
             f()
 
+        import inspect
+        abs_classes = [getattr(typing, attr) for attr in typing.__all__]
```

Comment:
Iterating over `typing.__all__` will only get you the public-API abstract classes. Iterating over `typing.__dict__.values()` might be more robust?

Reply:
Good idea, done.